feature/bgp/static-route_bgp_redistribution/README.md

```diff
@@ -0,0 +1,572 @@
+# RT-1.27: Static route to BGP redistribution
+
+## Summary
+
+- Static routes selected for redistribution base on combination of: prefix-set, set-tag
+- MED set to value of metric of static route (metric propagation)
+- AS-Path set to contain one AS with value provided in configuration
+- Local-Preference to a value provided in configuration
+- Community list set to defined community set
+- BGP protocol next-hop set to value provided in configuration
+- Redstribute static-route with "DROP" as the next-hop
+
+## Testbed type
+
+*   https://github.com/openconfig/featureprofiles/blob/main/topologies/atedut_4.testbed
+
+## Procedure
+
+#### Initial Setup:
+
+*   Connect DUT port-1, 2 and 3 to ATE port-1, 2 and 3 respectively
+
+*   Configure IPv4 and IPv6 addresses on DUT and ATE ports as shown below
+    *   DUT port-1 IPv4 address ```dp1-v4 = 192.168.1.1/30```
+    *   ATE port-1 IPv4 address ```ap1-v4 = 192.168.1.2/30```
+
+    *   DUT port-2 IPv4 address ```dp2-v4 = 192.168.1.5/30```
+    *   ATE port-2 IPv4 address ```ap2-v4 = 192.168.1.6/30```
+
+    *   DUT port-3 IPv4 address ```dp3-v4 = 192.168.1.9/30```
+    *   ATE port-3 IPv4 address ```ap3-v4 = 192.168.1.10/30```
+
+    *   DUT port-1 IPv6 address ```dp1-v6 = 2001:DB8::1/126```
+    *   ATE port-1 IPv6 address ```ap1-v6 = 2001:DB8::2/126```
+
+    *   DUT port-2 IPv6 address ```dp2-v6 = 2001:DB8::5/126```
+    *   ATE port-2 IPv6 address ```ap2-v6 = 2001:DB8::6/126```
+
+    *   DUT port-3 IPv6 address ```dp3-v6 = 2001:DB8::9/126```
+    *   ATE port-3 IPv6 address ```ap3-v6 = 2001:DB8::10/126```
+
+*   Create two IPv4 networks i.e. ```ipv4-network = 192.168.10.0/24``` and ```ipv4-drop-network = 192.168.20.0/24``` attached to ATE port-2
+
+*   Create two IPv6 networks i.e. ```ipv6-network = 2024:db8:128:128::/64``` and ```ipv6-drop-network = 2024:db8:64:64::/64``` attached to ATE port-2
```

Comment:
Ack, will update

Reply:
Since we resolved offline what the expectation is, you could explicitly state it in the subtest which does the redistribution of a static drop route.   (Next-hop should be equal to the DUT BGP IP address / ie: next-hop-self)